src/eventHandlers/checkSuite/index.spec.ts

```diff
@@ -334,8 +336,68 @@ describe('check Suite event handler', (): void => {
         },
       });
 
-    await checkSuiteHandle(octokit, 'dependabot-preview[bot]');
+    await checkSuiteHandle(octokit, 'dependabot-preview[bot]', 3);
 
     expect(warningSpy).toHaveBeenCalled();
   });
+
+  it('retries up to two times before failing', async (): Promise<void> => {
```

Comment:
A test is missing where you assert the backoff strategy.

Reply:
I have added one more check in the tests (see e849d10) to check whether the second retry will happen in the logged time. But I am not sure if this is what you mean. Otherwise, care to explain?

Reply:
Yes that's fine actually